backend/pipelined/filter.go

```diff
@@ -54,8 +46,10 @@ func evaluateEventFilter(event *types.Event, filter *types.EventFilter) bool {
 		}
 	}
 
+	synth := dynamic.Synthesize(event)
+
 	for _, statement := range filter.Statements {
-		match := evaluateEventFilterStatement(event, statement)
+		match := evaluateJSFilter(synth, statement, assets)
```

Comment:
I thought you said you were replacing the "statement" terminology with the "expression" terminology?